changes.md

```diff
@@ -0,0 +1,161 @@
+This file enumerates the changes that were made to the API surface between the Origin Trial as shipped in Chrome 83,
+and what is or will be available in Chrome 86.
+
+## File Picker API entry points
+
+**Before (in Origin Trial)**
+```javascript
+let file1 = await window.chooseFileSystemEntries(
+    {type: 'open-file'});
+let files = await window.chooseFileSystemEntries(
+    {type: 'open-file', multiple: true});
+let file2 = await window.chooseFileSystemEntries(
+    {type: 'save-file'});
+let dir = window.chooseFileSystemEntries(
+    {type: 'open-directory'});
+```
+**After (in Chrome M86)**
+```javascript
+let [file1] = await window.showOpenFilePicker();
+let files = await window.showOpenFilePicker({multiple: true});
+let file2 = await window.showSaveFilePicker();
+let dir = await window.showDirectoryPicker();
+```
+
+## Specifying accepted file types
+
+**Before (in Origin Trial)**
+```javascript
+await window.chooseFileSystemEntries({
+  accepts: [
+    {
+      description: 'Text Files',
+      mimeTypes: ['text/plain', 'text/html'],
+      extensions: ['txt', 'text', 'html', 'htm']
+    },
+    {
+      description: 'Images',
+      mimeTypes: ['image/*'],
+      extensions: ['png', 'gif', 'jpeg', jpg']
+    }
+  ]
+});
+```
+**After (in Chrome M86)**
```

Comment:
You might need a blank line before this for some Markdown processors.

(same for all code blocks below)